### DISCUSSION TITLE
Cant change nav bar APP_NAME

### DISCUSSION THREAD

**Discussion by andykoe24** [laravel/nova-issues#5497] [Q&A] [answered]
Hi,
i just upgrade nova v3 to v4.23.0.
i tried to change the app_name but it just wont change, keep showing 'Laravel Nova',
in config/nova.php, i do have this value in it
'name' => env('NOVA_APP_NAME', env('APP_NAME')),

i log it and it able to get the correct app_name, anyone any idea why this happen?
it working fine in v3.

![image](https://user-images.githubusercontent.com/112467629/234777140-f9d3da3a-f5e1-478d-97fb-383e1e685286.png)

**Comment by crynobone** [comment, marked as answer]
You need to change the brand logo, the text is part of the default SVG logo.
